src/main/java/com/loadero/http/TestRunTypeAdapter.java

```diff
@@ -0,0 +1,175 @@
+                    break;
+                case EXECUTION_STARTED:
+                    value = jsonReader.nextString();
+                    String executionStarted = Objects.isNull(value) ? "" : value;
+                    testRunMap.put(EXECUTION_STARTED, executionStarted);
+                    break;
+                case EXECUTION_FINISHED:
+                    value = jsonReader.nextString();
+                    String executionFinished = Objects.isNull(value) ? "" : value;
+                    testRunMap.put(EXECUTION_FINISHED, executionFinished);
+                    break;
+                case SCRIPT_FILE_ID:
+                    testRunMap.put(SCRIPT_FILE_ID, jsonReader.nextInt());
+                    break;
+                case TEST_NAME:
+                    testRunMap.put(TEST_NAME, jsonReader.nextString());
+                    break;
+                case START_INTERVAL:
+                    Duration startInterval = Duration.ofSeconds(jsonReader.nextInt());
+                    testRunMap.put(START_INTERVAL, startInterval);
+                    break;
+                case PARTICIPANT_TIMEOUT:
+                    Duration timeout = Duration.ofSeconds(jsonReader.nextInt());
+                    testRunMap.put(PARTICIPANT_TIMEOUT, timeout);
+                    break;
+                case LAUNCHING_ACCOUNT_ID:
+                    testRunMap.put(LAUNCHING_ACCOUNT_ID, jsonReader.nextInt());
+                    break;
+                case SUCCESS_RATE:
+                    testRunMap.put(SUCCESS_RATE, jsonReader.nextDouble());
+                    break;
+                case GROUP_COUNT:
+                    testRunMap.put(GROUP_COUNT, jsonReader.nextInt());
+                    break;
+                case PARTICIPANT_COUNT:
+                    testRunMap.put(PARTICIPANT_TIMEOUT, jsonReader.nextInt());
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        jsonReader.endObject();
+
+        double successRate = testRunMap.get(SUCCESS_RATE) == null ? 0.0 :
+            (double) testRunMap.get(SUCCESS_RATE);
+        int groupCount = testRunMap.get(GROUP_COUNT) == null ? 0 :
+            (int) testRunMap.get(GROUP_COUNT);
+        int participantCount = testRunMap.get(PARTICIPANT_COUNT) == null ? 0 :
+            (int) testRunMap.get(PARTICIPANT_COUNT);
+        String processingStarted = testRunMap.get(PROCESSING_STARTED) == null ? ""
+            : (String) testRunMap.get(PROCESSING_STARTED);
+        String processingFinished = testRunMap.get(PROCESSING_FINISHED) == null ? ""
+            : (String) testRunMap.get(PROCESSING_FINISHED);
+        String executionStarted = testRunMap.get(EXECUTION_STARTED) == null ? ""
+            : (String) testRunMap.get(EXECUTION_STARTED);
+        String executionFinished = testRunMap.get(EXECUTION_FINISHED) == null ? ""
+            : (String) testRunMap.get(EXECUTION_FINISHED);
```

Comment:
You can fit some of these in one line  and the one that do not fit 120 chars, wrap like this:
```java
        int participantCount = testRunMap.get(PARTICIPANT_COUNT) == null ? 0 : (int) testRunMap.get(PARTICIPANT_COUNT);
        String processingStarted = testRunMap.get(PROCESSING_STARTED) == null
                ? ""
                : (String) testRunMap.get(PROCESSING_STARTED);
````